srcpkgs/monero/template

```diff
@@ -43,6 +43,10 @@ case "$XBPS_TARGET_MACHINE" in
 	;;
 	ppc64*) configure_args+=" -DARCH=ppc64"
 	;;
+	ppc*) configure_arch+=" -DARCH=ppc"
+                makedepends+=" libatomic-devel"
+                CFLAGS+=" -latomic"
```

Comment:
Is that really `CFLAGS` and not `LDFLAGS`?

Reply:
Well, the ARM parts seem to be using `CFLAGS`, so probably really `CFLAGS`